Add bitmex scrapper

boyin0x opened the pull request:
Adds BitMex scraper implementation. 
Closes issue #516

nnn-gif reviewed (commented):
Getting error on websocket connection

BitMexScraper: Creating a new connection caused by err=websocket: close 1005 (no status)

boyin0x commented:
Hi @nnn-gif ,  there were too many expired symbols in the json file, I removed them and added a log warning when the scraper tries to subscribe and the symbol is invalid or expired.
